01_fib/fibonacci.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func calculateFib(n int) []int {
+	f0, f1, result := 1, 1, []int{}
+
+	for i := 1; i <= n; i++ {
+		result = append(result, f0)
+		f0, f1 = f1, f0+f1
+	}
+	return result
```

Comment:
I like how this is only calculating the sequence and not doing any IO (writing formatted output to stdout).